Bio/EBI/dbfetch.py

```diff
@@ -0,0 +1,82 @@
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+
+"""Provides code to access the EBI dbfetch tool.
+
+This module aims to make dbfetch easy to use. See:
+http://www.ebi.ac.uk/Tools/dbfetch/
+
+The dbfetch tool from EBI provvides access to different databases via a simple
+URL that the module constructs. Errors are handled regarding databse access and
+HTPP level.
+
+The functionality is somewhat similar to Biopython's Bio.TogoWS, Bio.KEGG.REST
+and Bio.Entrez modules.
+
+EBI provvides extensive guidelines for the databases, their format and the
+syntax necessary to use the tool. See:
+http://www.ebi.ac.uk/Tools/dbfetch/syntax.jsp
+
```

Comment:
Do we need to write that we do not support the SOAP services (  http://www.ebi.ac.uk/Tools/webservices/services/dbfetch ) in this comment?
That is written in the comment of TogoWS `__init__.py`.